other/materials_designer/specific_examples/defect_surface_adatom_graphene.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>We should explain that coordinates are in crystal units</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>